src/rendered_data_panel.ts

```diff
@@ -616,7 +617,9 @@ export abstract class RenderedDataPanel extends RenderedPanel {
     registerActionListener(element, "move-to-mouse-position", () => {
       const { mouseState } = this.viewer;
       if (mouseState.updateUnconditionally()) {
+        this.isMovingToMousePosition = true;
```

Comment:
as discussed in the meeting, it would be better for this to be more global than a single RenderedDataPanel